src/MHZ19.cpp

```diff
@@ -52,13 +54,13 @@ void MHZ19::begin(Stream &serial)
 
 void MHZ19::setRange(int range)
 {
-    if (range > 65000)
+    if (range > __INT_MAX__)
```

Comment:
This value is to do with the sensor and I'm not sure __INT_MAX__ will work as expected. Best to leave this as 65000.

Reply:
But you know, that an 16 bit int cannot be greater than 2^15 -1 (32767)?

Reply:
In the manual I found:
`Note: Detection range is 2000 or 5000ppm`
so the check is a bit random. Maybe 
`if (range > 500) { // safety net`
makes more sense?

Reply:
The lower threshold does appear to be around 2,000 ppm and the upper 10,000 ppm.

Perhaps the below would cover all use cases?  

if(range >= 500 && range <= 30000) 

20,000 would work as well.

Reply:
Of course, with respect to the error messages  :).

if(range < 500 || range > 30000)

Reply:
I took 20000 and commited, this looks more sensible 😀